src/main/java/lotto/Main.java

```diff
@@ -0,0 +1,33 @@
+package lotto;
+
+import lotto.model.LottoResult;
+import lotto.model.LottoTickets;
+import lotto.model.LottoWinningNumbers;
+import lotto.model.PurchaseAmount;
+import lotto.model.dto.LottoResultDto;
+import lotto.model.dto.LottoTicketDto;
+import lotto.view.Input;
+import lotto.view.Output;
+
+import java.util.stream.Collectors;
+
+public class Main {
+
+    public static void main(String[] args) {
+        PurchaseAmount purchaseAmount = new PurchaseAmount(Input.getPurchaseAmount());
+        LottoTickets lottoTickets = LottoController.buyLottoTickets(purchaseAmount);
+
+        Output.printPurchaseCount(lottoTickets.getSize());
+        Output.printLottoTickets(lottoTickets.getLottoTickets()
+                .stream()
+                .map(LottoTicketDto::new)
+                .collect(Collectors.toList()));
+
+        String winningNumbers = Input.getWinningNumbers();
+        int bonusNumber = Input.getBonusNumber();
+        LottoWinningNumbers lottoWinningNumbers = new LottoWinningNumbers(winningNumbers, bonusNumber);
+
+        LottoResult lottoResult = lottoTickets.getWinningResult(lottoWinningNumbers);
+        Output.printResult(new LottoResultDto(lottoResult));
```

Comment:
(1), (2) 로직도 각각 `LottoController` 안에 두고 LottoController의 public 메서드로 호출하여 결과를 얻어오도록 변경해보면 어떨까요? 

```suggestion
        // (1)
        String winningNumbers = Input.getWinningNumbers();
        int bonusNumber = Input.getBonusNumber();
        LottoWinningNumbers lottoWinningNumbers = new LottoWinningNumbers(winningNumbers, bonusNumber);

        // (2)
        LottoResult lottoResult = lottoTickets.getWinningResult(lottoWinningNumbers);
        Output.printResult(new LottoResultDto(lottoResult));
```

지금은 main 함수와 Controller의 로직이 서로 비슷한 느낌이네요. 🤔

아니면 `LottoController.play()` 같이 수행하고  LottoController의 play() 함수 안에서 지금 main 함수에 있는 로직들을 수행하도록 변경해도 괜찮을 것 같아요. 🙂

Reply:
반영하였습니다!